test/function/experimental/sum.yml

```diff
@@ -1,5 +1,6 @@
 name: "sum() function"
-experimental: true
+features:
+- experimentalFunctions
```

Comment:
Should these be `aggregateFunctions` instead since that is what they actually are?

Reply:
Sure!

Reply:
Actually, thinking about it, it's nice to be able to group experimental functions of any kind. If we add things like `pt::text()` that aren't official, they would also use `experimentalFunctions`. We shouldn't have too many of these feature flags, since they must be added in the test runner, too.